src/app/cdk/panel/panels/panels.component.ts

```diff
@@ -19,9 +24,33 @@ export class PanelsComponent implements OnInit {
   @Input() total
   @Input() isPublicRecord: any = false
 
-  constructor() {}
+  constructor(
+    private _dialog: MatDialog,
+    private _platform: PlatformInfoService
+  ) {}
+
+  add() {
+    switch (this.type) {
+      case 'peer-review':
+        this.openModal(ModalPeerReviewsComponent)
```

Comment:
@DanielPalafox Maybe the modal should be an input of the component? Similar to what we do on the app-component

Reply:
Ok